packages/lib-utils/src/app/api-discovery/index.ts

```diff
@@ -30,7 +30,9 @@ const pluralizeKind = (kind: string): string => {
 
 const defineModels = (list: APIResourceList): K8sModelCommon[] => {
   const { groupVersion } = list;
-  const [apiGroup, apiVersion] = groupVersion.split('/');
+  const [apiGroup, apiVersion] = `${groupVersion.includes('/') ? '' : '/'}${groupVersion}`.split(
+    '/',
+  );
```

Comment:
Alternatively:
```suggestion
  const [apiVersion, apiGroup] = groupVersion.split('/').reverse();
```

Reply:
Is it better? You gotta think about the reverse in a different way, whereas the current specifally says if no slash is found prepend it so we get an empty `apiGroup`.

Reply:
An `apiGroup` whose value is `''` or `undefined` is treated the same below where the value of `apiGroup` is read. Therefore the net result should be the same. Note that in my suggestion, the deconstruction assignment is reversed compared to the original line of code and `apiGroup` will be `undefined` if the split results in an array of length 1.

Reply:
@karelhala I tend to favor @christianvogt's suggestion for readability and simplicity. We could also set a default value for `apiGroup` if it turns out to be `undefined` if necessary.

Reply:
@florkbr `apiGroup` already has a default value. 

@christianvogt @florkbr if you prefer to read the `reverse` I am fine with it. For me it makes a bit more sense to "prepend" the string with `/` if not found. Both things are doing the same, it's just the first one is more straight forward, whereas the `reverse` with flipped destruct requires a bit more thinking of what it actually does.